v2/pages/index/cards.jsx

```diff
@@ -0,0 +1,37 @@
+import { useCollectionOnce } from 'react-firebase-hooks/firestore';
+import { firestore } from '../../api/firebase';
+import { Card } from '../../components/UI';
+
+const Cards = () => {
+  const [value, loading, error] = useCollectionOnce(firestore.collection('Opportunities').limit(3));
+  return (
+    <div className="card-group">
+      {!loading &&
+        value &&
+        value.docs.map((doc) => {
+          const { title, description } = doc.data();
+          return (
+            !loading && (
```

Comment:
This loading check is redundant, it doesn't change during iteration and it is checked in the condition above